### PR TITLE
install_zip.sh: convert sparse images

### PR DESCRIPTION
**Pull request by metti**
Convert *.img files that are detected as 'sparse' to regular image
files.

### CODE REVIEW

**Comment by Databean** [comment]
Can you please add a new version to debian/changelog?

debchange should make it easier to get the format right.

File: debian/changelog [hunk 1]
```diff
@@ -1,6 +1,14 @@
+cuttlefish-common (0.9+nmu5) UNRELEASED; urgency=medium
+
+  * install_zip.sh: exit early in case of errors
+  * install_zip.sh: support partial extraction
```
Comment (Databean) [inline debian/changelog]: I'm not clear on where partial extraction and exiting early is added. The 'find "${destdir}" -name "*.img"' code only seems to do the sparse image conversion.

Comment (metti) [reply debian/changelog]: That happened already here: https://github.com/google/android-cuttlefish/pull/6
This commit updates the change log including the missing entry from the mentioned PR.

Comment (Databean) [reply debian/changelog]: Sounds good, thanks.

**Pull request merged**